sdk/orbital/ci.yml

```diff
@@ -44,4 +38,3 @@ extends:
       - name: azure-resourcemanager-orbital
         groupId: com.azure.resourcemanager
         safeName: azureresourcemanagerorbital
-        releaseInBatch: ${{ parameters.release_azureresourcemanagerorbital }}
```

Comment:
This was set to avoid additional change when data-plane lib is added in future, as in foreseeable future there is no case when data-plane lib to be released together with mgmt-plane lib.